### ISSUE TITLE
Allow up to 3 VCs on dashboard

### ISSUE THREAD

**Issue by techsmyth** [alkem-io/client-web#6518]
## Description

As a user I want to be able to create and experiment with more than one VC

## Acceptance criteria

- [ ] Allow users on the VC campaign to have a maximum of 3 VCs
- [ ] Show up to 3 VCs on dashboard
- [ ] Provide feedback on dashboard if cannot create another VC

Note: the amount should be driven by licensing, but for the purpose of unblocking this story we start with 3.

## Additional Context

Screen shots / UX designs / data that are relevant.

## Areas that will be affected 
 
_To be added during the refinement_

**Comment by jeroentskui** [comment]
questions: 
how many spaces do we support?
how do we inform users of this 3 vc limit?

**Comment by bobbykolev** [comment]
### current
As discussed, for this version of the flow, we're supporting just one space and up to 3 VCs. This is valid for the MyAccount widget.
### future
In the future, we can make the space field a dropdown. 
### current
There's no information on the limit for now. From 1 we're bumping up to 3. 
### future
At some point, the amount of VCs will depend on the license.

**Comment by ccanos** [comment]
There's no real limit on the number of VCs shown, the button is hidden correctly but the limit doesn't apply if the user has more than 3 VCs: 
<img width="505" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNXkxQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--9dea9b3dc8432d0cef33e13859c2f97365dfc20f/image.png" alt="image.png" />

**Comment by techsmyth** [comment]
I think the above is correct, if there are more than 3 then they should for now just be shown - or we have to do something more intelligent like show 3 + then have see more etc. So I would suggest to just leave it as it is for now and close this issue - and then raise a separate issue to make the display of the number of VCs limited to avoid taking up too much screen real estate. Also the limit in terms of the button should later come from licensing - but what is above is enough.
